examples/experimental/wasm/server/api/sum.ts

```diff
@@ -1,7 +1,9 @@
-import { useQuery, lazyHandle } from 'h3'
+import { useQuery, defineLazyHandler } from 'h3'
```

Comment:
`useQuery` is auto-imported.

```suggestion
import { defineLazyHandler } from 'h3'
```